generators/server/templates/src/main/resources/config/_bootstrap.yml

```diff
@@ -0,0 +1,10 @@
+# ===================================================================
+# Spring Cloud Config bootstrap configuration
+# ===================================================================
+
+spring:
+    application:
+        name: <%= baseName %>
+    cloud:
+        config:
+            uri: http://localhost:8761/config
```

Comment:
For standalone mode, I found an example of a project using ZooKeeper, when it configured is [in standalone mode it uses a ZooKeeper in-memory instance](https://github.com/4finance/micro-infra-spring/blob/b5f234e32e0b325272d5cb9bbd615fedc092c0ea/micro-deps/micro-deps-spring-config/src/main/java/com/ofg/infrastructure/discovery/SpringCloudZookeeperConnectorConfiguration.java).

It makes me think that maybe in standalone mode (via a profile), we could either 
- disable Eureka and Config client
- configure the app to be its own Eureka server and Config server using application.yml as config store